lib/site_settings/label_formatter.rb

```diff
@@ -80,33 +80,41 @@ class LabelFormatter
       %w[youtube YouTube],
     ].freeze
 
+    HUMANIZED_ACRONYMS_HASH = HUMANIZED_ACRONYMS.map { |a| [a, true] }.to_h.freeze
```

Comment:
Instead of doing this, we can convert `HUMANIZED_ACRONYMS` to use a Set since `Set#include?` is O(1) complexity.